dev/diffs/3.5.1.diff

```diff
@@ -2120,29 +2120,17 @@ index 746f289c393..bc01ffd52ea 100644
        checkAnswer(aggDF, df1.groupBy("j").agg(max("k")))
      }
    }
-@@ -1013,7 +1039,9 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils with Adapti
-     }
-   }
- 
--  test("bucket coalescing is applied when join expressions match with partitioning expressions") {
-+  // https://github.com/apache/datafusion-comet/issues/617
-+  test("bucket coalescing is applied when join expressions match with partitioning expressions",
-+      IgnoreComet("TODO: fix Comet for this test")) {
-     withTable("t1", "t2", "t3") {
-       df1.write.format("parquet").bucketBy(8, "i", "j").saveAsTable("t1")
-       df2.write.format("parquet").bucketBy(4, "i", "j").saveAsTable("t2")
-@@ -1029,15 +1057,23 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils with Adapti
+@@ -1029,15 +1055,23 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils with Adapti
            Seq(true, false).foreach { aqeEnabled =>
              withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> aqeEnabled.toString) {
                val plan = sql(query).queryExecution.executedPlan
 -              val shuffles = collect(plan) { case s: ShuffleExchangeExec => s }
 +              val shuffles = collect(plan) {
-+            case s: ShuffleExchangeLike => s
-+          }
++                case s: ShuffleExchangeLike => s
++              }
```

Comment:
nit: this part of diff seems unnecessary

Reply:
this was just fixing incorrect formatting

Reply:
nm, I see what you mean now

Reply:
fixed